packages/playwright-ct-react/package.json

```diff
@@ -29,5 +29,9 @@
     "@vitejs/plugin-react": "^3.0.0",
     "@playwright/test": "1.30.0-next",
     "vite": "^4.0.3"
+  },
+  "peerDependencies": {
```

Comment:
We don't declare the peer dependencies, they don't really help with this use case.

Reply:
Where would I document that this library only works with a particular set of React versions?

Reply:
We would either need to support both 17 and 18, or fork the entire module to support 17 in packages/playwright-ct-react-17.

Reply:
Can't support both for the reasons outlined in the PR description. Going to create a fork.